js/freewheel.js

```diff
@@ -541,7 +541,7 @@ OO.Ads.manager(function(_, $) {
       // if (fwContext && _.isFunction(fwContext.dispose)) fwContext.dispose();
     };
 
-    var onResize = function() {
+    var updateOverlayPosition = function() {
       //Overlay placement issue - PBI-1227 as of 12/9/2015
       //The main issue with Freewheel is when notifying their SDK of video size changes,
       //Freewheel only attempts to resize ads and not re-position ads.
```

Comment:
I can rename these functions if there is some confusion. The change I made here was to pull out the logic for updating the overlay position into its own function (updateOverlayPosition) and call that inside onResize. I also call updateOverlayPosition when we get notified when the main video element comes into focus.